client/landing/stepper/declarative-flow/internals/steps-repository/domain-search/index.tsx

```diff
@@ -1,12 +1,25 @@
-import { DOMAIN_FLOW, Step, StepContainer } from '@automattic/onboarding';
+import { DOMAIN_FLOW, isHundredYearDomainFlow, Step, StepContainer } from '@automattic/onboarding';
 import { WPCOMDomainSearch } from 'calypso/components/domains/wpcom-domain-search';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { getSuggestionsVendor } from 'calypso/lib/domains/suggestions';
 import { shouldUseStepContainerV2 } from '../../../helpers/should-use-step-container-v2';
 import type { Step as StepType } from '../../types';
 
 import './style.scss';
 
 const DomainSearchStep: StepType = function DomainSearchStep( { flow } ) {
+	const config = {
+		vendor: getSuggestionsVendor( {
+			isSignup: false,
+			isDomainOnly: false,
+			flowName: flow,
+		} ),
+		priceRules: {
+			hidePrice: isHundredYearDomainFlow( flow ),
```

Comment:
```suggestion
			hidePrice: isHundredYearPlanFlow( flow ),
```
I think it should be `isHundredYearPlanFlow` here 🤔 we hide the price in the [hundred year plan flow](https://wordpress.com/setup/hundred-year-plan/domains), but show a one time price in the [hundred year domain flow](https://wordpress.com/setup/hundred-year-domain/domains).

Reply:
True